src/conduits/java/com/enderio/conduits/common/types/FluidExtendedData.java

```diff
@@ -70,7 +70,7 @@ public CompoundTag serializeGuiNBT() {
     public void deserializeNBT(CompoundTag nbt) {
         if (nbt.contains(ConduitNBTKeys.FLUID) && !isMultiFluid) {
             String fluid = nbt.getString(ConduitNBTKeys.FLUID);
-            if (fluid.equals("null")) {
+            if (fluid.equals("null") || fluid.equals("minecraft:empty")) {
```

Comment:
There is no guarantee that the string will be "minecraft:empty" (it could just be "empty") so I would use something like `new ResourceLocation(fluid).toString().equals("minecraft:empty")`